lambdas/data-migration2/src/granulesAndFiles.ts

```diff
@@ -20,15 +20,15 @@ import {
   PostgresUpdateFailed,
 } from '@cumulus/errors';
 
-import { MigrationSummary } from './types';
+import { GranuleDynamoSearchParams, MigrationResult } from '@cumulus/types/migration';
```

Comment:
🤔  It's out of scope for this ticket, but I'm wondering if we should consider making the granuleId search param a *list* of granuleIds, as if there are failures to target it would allow users to munge their failure results and batch submit a targeted list of failed granules.

Reply:
That's a good idea. I'll update

Reply:
Apparently you can’t do an IN/CONTAINS operator in a query against a Dynamo:

https://stackoverflow.com/a/32751240

So it’s not really easy to allow people to specify a list of granule IDs  for the migration without doing it via a scan, which then eliminates the efficiency benefit. 

It also seems like given the forthcoming speed improvements from CUMULUS-2468, asking people to re-run an entire set of granules via the collection may not be unreasonable. But I'm open to other ideas

Reply:
I think it's worth discussing but that clearly exceeds the scope needed for this ticket/change.